src/main/java/frc/robot/subsystems/endEffector/TOFSensor.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot.subsystems.endEffector;
+
+import com.playingwithfusion.TimeOfFlight;
+import com.playingwithfusion.TimeOfFlight.RangingMode;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class TOFSensor {
+  public double m_distance_EMA;
+  public double m_dist_SDEV_sq;
+  public String mode = "Short";
+
+  int m_rangeX0;
+  int m_rangeY0;
+  int m_rangeX1;
+  int m_rangeY1;
+
+  private TimeOfFlight sensor;
+
+  public TOFSensor(int CAN_ID) {
+    sensor = new TimeOfFlight(CAN_ID);
+    RoI(0, 0, 15, 15);
+    sensor.setRangingMode(RangingMode.Short, 0.24); // Will need to test. must be between 24-1000ms
```

Comment:
The units for sampling time is milliseconds and 24 is the minimum https://www.playingwithfusion.com/frc/2025/javadoc/com/playingwithfusion/TimeOfFlight.html#setRangingMode(com.playingwithfusion.TimeOfFlight.RangingMode,double). Based on my interpretation of the datasheet it looks like the main benefit of increasing sampling time is power conservation which is not a major concern for us.

So this should look like
`sensor.setRangingMode(RangingMode.Short, 24);`

Reply:
hm interesting. i didn't notice that actually